test/Bitreserve/Tests/Model/TransactionTest.php

```diff
@@ -244,7 +244,7 @@ public function shouldCancel()
      * @test
      * @expectedException \Bitreserve\Exception\ErrorException
      */
-    public function shouldCancelThownAnErrorWhenStatusIsNotPending()
+    public function shouldCancelThownAnErrorWhenStatusIsNotWaiting()
```

Comment:
Typo in all methods (thown -> throw). Also, I think the wording is not the best when using should. (cancelShould is also not nice..).